cmd/local/uninstall.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func NewUninstallCmd() *cobra.Command {
-
+	var purge = false
```

Comment:
The purge var already gets a default false value from the flag 
```suggestion
	var purge
```

Reply:
'var purge bool'